packages/odyssey/src/scss/vendors-ext/_choices-ext.scss

```diff
@@ -0,0 +1,453 @@
+$choices-icon-cross-inverse: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjEiIGhlaWdodD0iMjEiIHZpZXdCb3g9IjAgMCAyMSAyMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48ZyBmaWxsPSIjMDAwIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPjxwYXRoIGQ9Ik0yLjU5Mi4wNDRsMTguMzY0IDE4LjM2NC0yLjU0OCAyLjU0OEwuMDQ0IDIuNTkyeiIvPjxwYXRoIGQ9Ik0wIDE4LjM2NEwxOC4zNjQgMGwyLjU0OCAyLjU0OEwyLjU0OCAyMC45MTJ6Ii8+PC9nPjwvc3ZnPg==') !default;
+
+.ods-select {
+  position: relative;
+  max-width: $max-line-length;
+  overflow: hidden;
+  font-size: ms(0);
+
+  &:focus,
+  &.is-ods-select-focused {
+    border-radius: $base-border-radius;
+    outline: none;
+    box-shadow: 0 0 0 4px fauxpacity(cv('action', 'base'));
+
+    &.is-ods-select-open {
+      box-shadow: 0 0 0 4px fauxpacity(cv('action', 'base')), 0 2px 0 4px fauxpacity(cv('action', 'base'));
+
+      &.is-ods-select-flipped {
+        box-shadow: 0 0 0 4px fauxpacity(cv('action', 'base')), 0 -2px 0 4px fauxpacity(cv('action', 'base'));
+      }
+    }
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  &.is-ods-select-open {
+    overflow: initial;
+  }
+
+  &.is-ods-select-disabled {
+    .ods-select--inner,
+    .ods-select--input {
+      border-color: cv('gray', '200');
+      background: cv('gray', '000');
+      color: $text-sub;
+      cursor: not-allowed;
+      user-select: none;
+
+      &:hover {
+        border-color: cv('gray', '200');
+      }
+    }
+
+    &:focus,
+    &.is-ods-select-focused {
+      box-shadow: none;
+    }
+
+    .ods-select--item {
+      cursor: not-allowed;
+    }
+  }
+
+  [data-invalid] & {
+    &:focus,
+    &.is-ods-select-focused {
+      border-radius: $base-border-radius;
+      outline: none;
+      box-shadow: 0 0 0 4px fauxpacity(cv('danger', 'base'));
+
+      &.is-ods-select-open {
+        box-shadow: 0 0 0 4px fauxpacity(cv('danger', 'base')), 0 2px 0 4px fauxpacity(cv('danger', 'base'));
+
+        &.is-ods-select-flipped {
+          box-shadow: 0 0 0 4px fauxpacity(cv('danger', 'base')), 0 -2px 0 4px fauxpacity(cv('danger', 'base'));
+        }
+      }
+    }
+
+    .ods-select--inner,
+    .ods-select--input {
+      border-color: cv('danger');
+    }
+  }
+
+  [hidden] {
+    display: none !important;
+  }
+}
+
+.ods-select[data-type*='select-one'] {
+  .ods-select--inner {
+    padding-bottom: $tiny-spacing;
+  }
+
+  .ods-select--input {
+    display: block;
+    width: 100%;
+    margin: 0;
+    padding: $small-spacing;
+    border-bottom: 1px solid $base-border-color;
+    background-color: $white;
+  }
+
+  .ods-select--button {
+    position: absolute;
+    top: 50%;
+    right: 0;
+    width: 20px;
+    height: 20px;
+    margin-top: -$small-spacing;
+    margin-right: 25px;
+    padding: 0;
+    border-radius: 10em;
```

Comment:
Double check here - is 10em intentional for a border radius?

Reply:
This is for an as-yet unused feature. We make it clear in the docs that the only Choices.js features we're supporting are those offered by a regular HTML `select` right now.

As we turn things on, we'll also tune these styles.